### PR TITLE
fix: low version of mongo not support retryWrites setting.

### PR DESCRIPTION
**Pull request by onebite**
Motivation:

provide a way to set retryable settings

### CODE REVIEW

File: src/main/java/io/vertx/ext/mongo/impl/config/MongoClientOptionsParser.java
Comment (tsegismont) [inline src/main/java/io/vertx/ext/mongo/impl/config/MongoClientOptionsParser.java:13]:
```suggestion
```

File: src/main/java/io/vertx/ext/mongo/impl/config/MongoClientOptionsParser.java
Comment (tsegismont) [inline src/main/java/io/vertx/ext/mongo/impl/config/MongoClientOptionsParser.java:17]:
```suggestion
```

File: src/main/java/io/vertx/ext/mongo/impl/config/MongoClientOptionsParser.java
Comment (tsegismont) [inline src/main/java/io/vertx/ext/mongo/impl/config/MongoClientOptionsParser.java:25]:
```suggestion
```

File: src/main/java/io/vertx/ext/mongo/impl/config/MongoClientOptionsParser.java
Comment (tsegismont) [inline src/main/java/io/vertx/ext/mongo/impl/config/MongoClientOptionsParser.java:105]:
```suggestion
      Boolean retryWrites = config.getBoolean("retryWrites");
      Boolean retryReads = config.getBoolean("retryReads");
```

File: src/main/java/io/vertx/ext/mongo/impl/config/MongoClientOptionsParser.java
Comment (tsegismont) [inline src/main/java/io/vertx/ext/mongo/impl/config/MongoClientOptionsParser.java:122]:
```suggestion
```

**Review by tsegismont — APPROVED** [review]
Thanks for the updates.

It looks good to me, except for the boolean value parsing. When the config comes from a `JsonObject`, we'll stick to "standard" JSON boolean values.

**Comment by onebite** [comment]
thank you for your patience. all suggestions have submit.

**Comment by tsegismont** [comment]
You're welcome. I'm waiting for the CI job result before merging.

**Pull request merged**
